source/portable/NetworkInterface/xilinx_ultrascale/NetworkInterface.c

```diff
@@ -348,12 +348,19 @@ static BaseType_t xUltrascaleNetworkInterfaceInitialise( NetworkInterface_t * px
                     XEmacPs_SetHash( pxEMAC_PS, ( void * ) xLLMNR_MacAddressIPv6.ucBytes );
                 }
                 #endif /* if ( ipconfigUSE_IPv6 == 0 ) */
+
+
+                /* allow reception of multicast addresses programmed into hash */
+                XEmacPs_SetOptions( pxEMAC_PS, XEMACPS_MULTICAST_OPTION );
```

Comment:
These are also missing in the Zynq7000 port and most likely needed.
Should I add it? But I cannot test it.